### PR TITLE
Darc. Remote and BAR querying

### PR DESCRIPTION
**Pull request by jcagme**
This is the first of several PRs related to Darc. 

This one contains the functionality to execute git commands in repos without the need of having them cloned.

Also, exposing some functions to interact with the Build Asset Registry

### CODE REVIEW

File: src/Microsoft.DotNet.Darc/src/DarcLib/Microsoft.DotNet.DarcLib.csproj [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
+	</PropertyGroup>
```
Comment (alexperovich) [inline src/Microsoft.DotNet.Darc/src/DarcLib/Microsoft.DotNet.DarcLib.csproj]: More tabs

File: src/Microsoft.DotNet.Github.IssueLabeler/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (alexperovich) [inline src/Microsoft.DotNet.Github.IssueLabeler/Properties/launchSettings.json]: Remove and ignore this file please.

File: src/Maestro/MaestroApplication/MaestroApplication.sfproj [hunk 3]
```diff
@@ -1,51 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (alexperovich) [inline src/Maestro/MaestroApplication/MaestroApplication.sfproj]: Please put this project back. It needs to exist.

**Comment by jcagme** [comment]
@dotnet-bot test prtest/Windows/NT/Release please

**Pull request merged**